Other/MergeShops/EctocaveMerge.cs

```diff
@@ -0,0 +1,198 @@
+                    break;
+                #endregion
+
+                case "Ichorus Scythe Piece":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(3874);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ectocave", "Ichor Dracolich", "Uncut Emerald", 9, log: false);
+                        Core.HuntMonster("ectocave", "Ektorax", "Brilliant Diamond", quant, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Ichorus Egg":
+                    if (!Bot.Player.IsMember)
+                        break;
+
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ectocave", "Ektorax", "Ichorus Egg", quant, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Pure Ichor Gem":
+                    if (!Bot.Player.IsMember)
+                        break;
+
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(3873);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ectocave", "Ichor Draconian", "Uncut Ichor Gem", 50, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Slime":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ectocave", "Swamp Lurker", "Slime", 20, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Dragon Rogue":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ectocave", "Ektorax", "Dragon Rogue", quant, log: false);
```

Comment:
any case with only this in the while can be made into a grouping case meanin

case"":
case"":
while----
dothing
Bot.Wait.ForPickup(req.Name);
break;

Reply:
as long as the mob is teh same and its only a hunt for the item from teh same mob for all the items, you can sl ojust cahnge the item to a `req.Name`